src/loaders/lottie/tvgLottieLoader.cpp

```diff
@@ -349,7 +349,10 @@ uint32_t LottieLoader::gen(const char* slots, bool byDefault)
         auto found = false;
         ARRAY_FOREACH(p, comp->slots) {
             if ((*p)->sid != sid) continue;  //find target
-            if (auto prop = parser.parse(*p)) custom->props.push({prop, *p});
+            if (auto prop = parser.parse(*p)) {
+                prop->sid = sid;
+                custom->props.push({prop, *p});
```

Comment:
LottieSlot already has a `sid`, maybe you can assign `sid` inside parser.parse().